README.md

```diff
@@ -93,9 +93,8 @@ Building the ISO:
 -
 Trigger the build by executing:
 ```
-pacman -Sy archiso
+sudo pacman -Sy archiso
```

Comment:
[don't use `pacman -Sy`](https://wiki.archlinux.org/title/System_maintenance#Partial_upgrades_are_unsupported)

Reply:
@getchoo What is a proper way? `pacman -Syu`?

Reply:
nope just `pacman -S`